k0776243/Midterm.py

```diff
@@ -0,0 +1,80 @@
+# Skeletons invade pirate ship.  Hero is stuck in bottom of ship and has to escape, get off the boat and to the island.  
+from sys import exit
+from random import randint
+
+class Ship(object):
+    
+    def enter(self)
+        print "You're stuck in the bottom of the ship and it's filling rapidly with water."
+        print "Choose between: a crowbar, the trapdoor key, or broccoli."
+    
+    action = raw_input("> ")
+    
+    if action == "crowbar":
```

Comment:
These lines need to be indented in order to be part of the enter method, otherwise they will run whenever you create a new ship, but you want them to run when we say `current_scene.enter()`